openassessment/xblock/static/js/spec/lms/oa_peer.js

```diff
@@ -74,6 +67,17 @@ describe("OpenAssessment.PeerView", function() {
         expect(server.peerAssess).toHaveBeenCalledWith(
             optionsSelected, criterionFeedback, overallFeedback, ''
         );
+    };
+
+    beforeEach(function() {
+        // Create a new stub server
+        server = new StubServer();
+        server.renderLatex = jasmine.createSpy('renderLatex');
+    });
+
+    it("Sends a peer assessment to the server", function() {
```

Comment:
Minor nit: I like Jasmine tests to read like sentences starting with "it", so I lowercase the first letter.